source/utilities.cc

```diff
@@ -2149,8 +2149,8 @@ namespace aspect
       time_dependent = false;
       // Give warning if first processor
       this->get_pcout() << std::endl
-                        << "   Loading new data file did not succeed." << std::endl
-                        << "   Assuming constant boundary conditions for rest of model run."
+                        << "   From this timestep onwards, ASPECT will not attempt to load new Ascii data files." << std::endl
+                        << "   If the Ascii data represented a time-dependent boundary condition, that dependence ends at this timestep."
```

Comment:
Maybe add something like "..., i.e., the boundary condition will simply continue unchanged from the last known state into the future"?

Reply:
Done :)